lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -499,28 +503,18 @@ data ApiNetworkParameters = ApiNetworkParameters
     , decentralizationLevel :: !(Quantity "percent" Percentage)
     } deriving (Eq, Generic, Show)
 
-toApiNetworkParameters :: BlockchainParameters -> ApiNetworkParameters
-toApiNetworkParameters bp = ApiNetworkParameters
-    (ApiT $ getGenesisBlockHash bp)
-    (ApiT $ getGenesisBlockDate bp)
-    (Quantity $ unSlotLength $ getSlotLength bp)
-    (Quantity $ unEpochLength $ getEpochLength bp)
-    (getEpochStability bp)
+toApiNetworkParameters :: NetworkParameters -> ApiNetworkParameters
+toApiNetworkParameters (NetworkParameters gp pps) = ApiNetworkParameters
```

Comment:
`pps` should probably be renamed to `pp`.

We generally don't append the `s` suffix to identifiers that represent values of types ending in `Parameters`.

Reply:
Fixed in 3ac0a59db5085f5aea98cdc26b0441b7fa82cf31.